[INFO   ] Logger: Record log in /Users/goncalo/.kivy/logs/kivy_19-06-12_25.txt
[INFO   ] Kivy: v1.10.1
[INFO   ] Python: v3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 16:52:21) 
[Clang 6.0 (clang-600.0.57)]
[INFO   ] Factory: 194 symbols loaded
[INFO   ] Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <gl>
[INFO   ] GL: OpenGL version <b'2.1 INTEL-12.9.22'>
[INFO   ] GL: OpenGL vendor <b'Intel Inc.'>
[INFO   ] GL: OpenGL renderer <b'Intel Iris OpenGL Engine'>
[INFO   ] GL: OpenGL parsed version: 2, 1
[INFO   ] GL: Shading version <b'1.20'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 249, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/Users/goncalo/Test/kivy/test.kv", line 7, in <module>
[WARNING] stderr:     rgba: self.rgba
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 30, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: AttributeError: 'Player' object has no attribute 'rgba'
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 843, in _build_canvas
[WARNING] stderr:     key, value, prule, idmap, True)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 254, in create_handler
[WARNING] stderr:     cause=tb)
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/Users/goncalo/Test/kivy/test.kv", line 7:
[WARNING] stderr: ...
[WARNING] stderr:       5:    canvas:
[WARNING] stderr:       6:        Color:
[WARNING] stderr: >>    7:            rgba: self.rgba
[WARNING] stderr:       8:        Rectangle:
[WARNING] stderr:       9:            pos: self.pos
[WARNING] stderr: ...
[WARNING] stderr: AttributeError: 'Player' object has no attribute 'rgba'
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 249, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/Users/goncalo/Test/kivy/test.kv", line 7, in <module>
[WARNING] stderr:     rgba: self.rgba
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 30, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: 
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "test.py", line 28, in <module>
[WARNING] stderr:     TestApp().run()
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/app.py", line 800, in run
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "test.py", line 25, in build
[WARNING] stderr:     return Test()
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/uix/widget.py", line 348, in __init__
[WARNING] stderr:     Builder.apply(self, ignored_consts=self._kwargs_applied_init)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 469, in apply
[WARNING] stderr:     self._apply_rule(widget, rule, rule, ignored_consts=ignored_consts)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 584, in _apply_rule
[WARNING] stderr:     self.apply(child)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 469, in apply
[WARNING] stderr:     self._apply_rule(widget, rule, rule, ignored_consts=ignored_consts)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 524, in _apply_rule
[WARNING] stderr:     rule.canvas_root, rootrule)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 849, in _build_canvas
[WARNING] stderr:     '{}: {}'.format(e.__class__.__name__, e), cause=tb)
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/Users/goncalo/Test/kivy/test.kv", line 7:
[WARNING] stderr: ...
[WARNING] stderr:       5:    canvas:
[WARNING] stderr:       6:        Color:
[WARNING] stderr: >>    7:            rgba: self.rgba
[WARNING] stderr:       8:        Rectangle:
[WARNING] stderr:       9:            pos: self.pos
[WARNING] stderr: ...
[WARNING] stderr: BuilderException: Parser: File "/Users/goncalo/Test/kivy/test.kv", line 7:
[WARNING] stderr: ...
[WARNING] stderr:       5:    canvas:
[WARNING] stderr:       6:        Color:
[WARNING] stderr: >>    7:            rgba: self.rgba
[WARNING] stderr:       8:        Rectangle:
[WARNING] stderr:       9:            pos: self.pos
[WARNING] stderr: ...
[WARNING] stderr: AttributeError: 'Player' object has no attribute 'rgba'
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 249, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/Users/goncalo/Test/kivy/test.kv", line 7, in <module>
[WARNING] stderr:     rgba: self.rgba
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 30, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: 
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 843, in _build_canvas
[WARNING] stderr:     key, value, prule, idmap, True)
[WARNING] stderr:   File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/kivy/lang/builder.py", line 254, in create_handler
[WARNING] stderr:     cause=tb)
[WARNING] stderr: 
